webdriver-spec.html

```diff
@@ -5799,7 +5816,7 @@
 <p>The <a>remote end steps</a> are:
 
 <ol>
- <li><p>If the <a>current top-level browsing context</a> is <a>no longer open</a>,
+ <li><p>If the <a>current browsing context</a> is <a>no longer open</a>,
```

Comment:
I guess cookies are tied to domain, and a frame may be on a different domain than the outer document.